From 3c733290e84e2878594167e6dd5bdcc1e3ecb2f3 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Wed, 30 Nov 2022 16:15:47 +0100
Subject: [PATCH 284/529] ARM: configs: Enable CPU_FREQ in multi_v7 fragments

CPU_FREQ flag was initially disabled in cleanup fragment.
This commits enable also arm scmi cpufreq driver.

Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-by: Alexandre TORGUE <alexandre.torgue@foss.st.com>
Change-Id: I12c896e670d3d2cb13aa8d5d158fe0833ecd7d0a
Reviewed-on: https://gerrit.st.com/c/mpu/oe/st/linux-stm32/+/279692
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Eric FOURMONT <eric.fourmont-ext@st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
---
 arch/arm/configs/fragment-01-multiv7_cleanup.config | 1 -
 arch/arm/configs/fragment-02-multiv7_addons.config  | 1 +
 2 files changed, 1 insertion(+), 1 deletion(-)

--- a/arch/arm/configs/fragment-01-multiv7_cleanup.config
+++ b/arch/arm/configs/fragment-01-multiv7_cleanup.config
@@ -46,7 +46,6 @@
 # CONFIG_CAN_MCP251X is not set
 # CONFIG_CAN_RCAR is not set
 # CONFIG_CHROME_PLATFORMS is not set
-# CONFIG_CPU_FREQ is not set
 # CONFIG_DETECT_HUNG_TASK is not set
 # CONFIG_EFI is not set
 # CONFIG_ENABLE_MUST_CHECK is not set
--- a/arch/arm/configs/fragment-02-multiv7_addons.config
+++ b/arch/arm/configs/fragment-02-multiv7_addons.config
@@ -6,6 +6,7 @@ CONFIG_ARM_SMC_MBOX=y
 CONFIG_ARM_STM32_CPUIDLE=y
 CONFIG_CHR_DEV_SG=y
 CONFIG_COMMON_CLK_SCMI=y
+CONFIG_CPU_FREQ=y
 CONFIG_CPU_IDLE=y
 CONFIG_CRC_ITU_T=m
 CONFIG_DEBUG_INFO=y
